src/pymor/analyticalproblems/expressions.py

```diff
@@ -181,10 +183,25 @@ def wrapper(*args, mu={}):
 
         return wrapper
 
+    def to_fenics(self, mesh, variable='x'):
+        # sanity check for dolfin before running code
+        config.require('FENICS')
+        from dolfin import Constant
+        from ufl import SpatialCoordinate
+        assert variable not in self.parameters or self.parameters[variable] == mesh.topology().dim()
```

Comment:
I did not understand where self.parameters are set, could you explain that (in a comment)?

Reply:
An Expression is a `ParametricObject` ...